src/content/faq/faq.json

```diff
@@ -0,0 +1,21 @@
+{
+    "faqs": [
+        {
+            "question": "how do I join?",
+            "answer": "answer answer answer answer answer answer  answer  answer  answer"
```

Comment:
LOL

Reply:
I thought it made sense for this bc there are multiple questions, do you think I should rename the file/component to "faqs" to match